docker-compose/setup-script/noco.sh

```diff
@@ -213,7 +248,20 @@ else
     message_arr+=("Watchtower: Disabled")
 fi
 
+echo "Show Advanced Options [Y/N] (default: N): "
+read -r ADVANCED_OPTIONS
+
+if [ -n "$ADVANCED_OPTIONS" ] && { [ "$ADVANCED_OPTIONS" = "Y" ] || [ "$ADVANCED_OPTIONS" = "y" ]; }; then
+    NUM_CORES=$(nproc)
```

Comment:
Does `nproc` command available in macos ?

<img width="362" alt="image" src="https://github.com/nocodb/nocodb/assets/61551451/5eb3079d-f9ce-4e59-b5e6-0cfb12d13b84">

If we are not planning to support Macos then it's fine (cc @o1lab).

Reply:
ya we wont need it on local machines